modules/loader/src/main/java/com/opengamma/strata/loader/csv/FxSingleBarrierOptionTradeCsvPlugin.java

```diff
@@ -97,7 +97,8 @@ private FxSingleBarrierOptionTrade parse(CsvRow row, TradeInfo info, TradeCsvInf
 
     return FxSingleBarrierOptionTrade.builder()
         .product(productBuilder.build())
-        .premium(premium)
+        .premium(CsvLoaderUtils.tryParsePremiumFromDefaultFields(row)
+            .orElse(AdjustablePayment.of(USD, 0d, vanillaOptionTrade.getProduct().getExpiryDate())))
```

Comment:
This currency should be one of the two in the FX. I suggest it is `currencyPair.toConventional().getBaseCurrency()` (Hard coding USD may force additional market data to be loaded that we don't want.)